### ISSUE TITLE
Error when launching

### ISSUE THREAD

**Issue by imrecsoka** [DGCDev/digitalcoin#38]
digitalcoin wallet  error st13runtimeerror  
no idea why

Windows 7 64 bit Ultimate   - Visual Runtimes for C++ are installed

![image](https://user-images.githubusercontent.com/25494601/27987801-41491112-6414-11e7-9c97-f8b343a11abd.png)

**Comment by Dreadhead86** [comment]
Im having the same problem with windows 10 and windows 7.

**Comment by imrecsoka** [comment]
why do these guys release shit code that is never tested properly.

**Comment by Dreadhead86** [comment]
yeah i have coins on this wallet aswell i cant access till this runtime error is fixed. all my other altcoin qt wallets work perfectly. I had no trouble with digitalcoins v2 wallets.

**Comment by imrecsoka** [comment]
Hi, can you maybe suggest me a coin to try and mine? have u solo mined any ?

**Comment by feanorfelagund** [comment]
Same issue tested on windows 10/7 and OSX 10.6

**Comment by ubronan** [comment]
Well apparantly nobody cares that people have coins in their wallet.
And no longer can access them because the v3.xx wallets crash instant at start.
even tried to find someone on irc, but as usual no responce at all

**Comment by imrecsoka** [comment]
I have left this wallet since there is no bloody community help nor support from the developers..... from what it looks here at least?

**Comment by ddombrowsky** [comment]
I can still run a command-line client, but as far as I know no exchanges support DGC any longer.  It's all but a dead coin.

**Comment by imrecsoka** [comment]
Has anybody sorted out the crashing error message when the wallet is launching ?

**Comment by lomtax** [comment]
Hi,

I took a look at the source code and the startup crash seems pretty easy to fix. 
I've got a working wallet on linux and osx, is someone interrested in the windows build ? 
I can fork the official source on github and store the new release on it if you want.

**Comment by Xandrah** [comment]
This can be fixed by adding to your hosts file 127.0.0.1 bittrex.com

This is because the compiled digitalcoin wallet still trys to get trade data from bittrex.com, support was killed off for DGC on bittrex.

**Comment by lomtax** [comment]
We have fixed the issue https://github.com/lomtax/digitalcoin and released a new wallet https://github.com/lomtax/digitalcoin/releases. If your are interested in you can join us on bitcointalk https://bitcointalk.org/index.php?topic=785601.780 or on discord https://discord.gg/UagVzAx.
